src/TrackingOptOut.php

```diff
@@ -115,6 +115,10 @@ public static function enqueueScriptsFirstHead() {
   public static function wp_enqueue_scripts() {
     wp_enqueue_script(Plugin::PREFIX . '-enable-opt_out', Plugin::getBaseUrl() . '/dist/scripts/enable-opt-out.js', ['jquery'], FALSE, TRUE);
     wp_localize_script(Plugin::PREFIX . '-enable-opt_out', 'core_standards', ['opt_out_confirmation_message' => __('Google Analytics tracking has been disabled.', Plugin::L10N)]);
+
+    wp_enqueue_script('ga_optout', plugins_url() . '/core-standards/dist/scripts/google-analytics-opt-out.js'); 
+    $cookiedomain = array('domainValue' => COOKIE_DOMAIN);
+    wp_localize_script('ga_optout', 'cookieDomain', $cookiedomain);
```

Comment:
1. The first argument registers the script with a custom identifier within WordPress.  It needs to start with the plugin's namespace.  See line 116/117 above.

2. The second argument is the name of the variable that will be exposed to the window/DOM document.  It needs to be the JavaScript-compatible variation of the plugin's namespace.  See line 117 above.

3. A better property name would be `cookieDomain` instead of `domainValue`.  The separate `$cookiedomain` variable does not seem to be necessary, as the array can be inlined.  And the array declaration should use the new short syntax.

4. The final cookie domain value including leading dot should be prepared here already: `['cookieDomain' => '.' . ltrim(COOKIE_DOMAIN, '.')]`